CONTRIBUTING.md

```diff
@@ -0,0 +1,272 @@
+- [Question or Problem?](#question)
+- [Issues and Bugs](#issue)
+- [Feature Requests](#feature)
+- [Submission Guidelines](#submit)
+- [Coding Rules](#rules)
+- [Pull Request Guidelines](#submit-pr)
+- [Signing the CLA](#cla)
+
+## <a name="coc"></a> Code of Conduct
+
+Help us keep Logseq open and inclusive.
+Please read and follow our [Code of Conduct][coc].
+
+## <a name="question"></a> Got a Question or a Problem?
+
+Do not open issues for general support questions or feature requests as we want to keep GitHub issues for bug reports.
+Instead, we recommend using [Logseq forum][forum] to ask support-related questions.
+
+The Logseq forum is a much better place to ask questions since:
+
+- there are more people willing to help on the forum
+- questions and answers stay available for public viewing so your question/answer might help someone else
+- The forum's voting system assures that the best answers are prominently visible.
+
+To save your and our time, we will systematically close all issues that are requests for general support and redirect people to the forum.
+
+If you would like to chat about the question in real-time, you can reach out via [our Discord server][discord].
+
+## <a name="issue"></a> Found a Bug?
+
+If you find a bug in the source code, you can help us by [submitting an issue](#submit-issue) to our [GitHub Repository][github].
+Even better, you can [submit a Pull Request](#submit-pr) with a fix.
+
+## <a name="feature"></a> Missing a Feature?
+
+You can *request* a new feature by [Creating a thread][feature-request] in our forum.
+If you would like to *implement* a new feature, please open an issue and outline your proposal so that it can be discussed. This process allows us to better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
+
+## <a name="submit"></a> Submission Guidelines
+
+### <a name="submit-issue"></a> Submitting an Issue
+
+Before you submit an issue, please search the [issue tracker][issue-tracker]. An issue for your problem might already exist and the discussion might inform you of workarounds readily available.
+
+Please include the following with each issue:
+
+- Version of Logseq
+- Your operating system
+- List of extensions that you have installed. Attempt to recreate the issue after disabling all extensions.
+- Reproducible steps (1... 2... 3...) that cause the issue
+- What you expected to see, versus what you actually saw
+- Images, animations, or a link to a video showing the issue occurring
+- A code snippet that demonstrates the issue or a link to a  code repository the developers can easily pull down to recreate the  issue locally
+  - **Note:** Because the developers need to copy and paste the code snippet, including a code snippet as a media file (i.e. .gif)  is not sufficient.
+- Errors from the Dev Tools Console (open from the menu: View > Toggle Developer Tools or press CTRL + Shift + i)
```

Comment:
Optional: This link may be helpful, as @cnrpman usually uses it to guide people that have issues.

https://help.figma.com/hc/en-us/articles/360041949073-Use-Chrome-Developer-Tools

Reply:
That's a good guide. This is what @logseq-cldwalker was asking about.

>  Optional: It could be nice to just point to the template link and then use this section to elaborate on things that aren't in the templates like the Dev Tools Console or tools in the new bug report page

I do not link to external 3rd parties. I prefer that links are either from Logseq.com or GitHub.com. 

It's best we can create our own document with instructions on using Dev tools and other tools that can be helpful in diagnosing issues.

For example; we can add a section on diagnosing performance issues, etc..

We can use this guide as the base for ours.